rootfs/etc/profile.d/terraform.sh

```diff
@@ -1,8 +1,7 @@
 PROMPT_HOOKS+=("terraform_prompt")
 function terraform_prompt() {
-	shopt -s nullglob
-	TF_FILES=(*.tf)
-	if [ ! -z "${TF_FILES}" ]; then
+	# Test if there are any files with names ending in ".tf"
+	if compgen -G '*.tf' >/dev/null; then
```

Comment:
a test for `*.tf.json` files would also seem salient here

- https://github.com/hashicorp/terraform/blob/21ba14ba1c93ca6ff087051c511601b355f9b47a/website/docs/configuration/syntax-json.html.md